apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -49,22 +81,426 @@ export const getPageInfo: GetPageNavInfo = ({
+                {intl.formatMessage({
+                  defaultMessage:
+                    "You’ve answered the following screening questions:",
+                  id: "GJc/j5",
+                })}
+              </p>
+              <ul>
+                {screeningQuestionResponses.map((response) => (
+                  <li key={response.id} data-h2-margin-bottom="base(x1)">
+                    <p
+                      data-h2-font-weight="base(700)"
+                      data-h2-margin-bottom="base(x.5)"
+                    >
+                      {response.screeningQuestion?.question
+                        ? response.screeningQuestion.question[locale]
+                        : ""}
+                    </p>
+                    <p>{response.answer}</p>
+                  </li>
+                ))}
+              </ul>
+            </div>
+          ) : (
+            <Well>
+              <p data-h2-text-align="base(center)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "<strong>It looks like you haven't answered any screening questions yet.</strong>",
+                  id: "Zx7wZQ",
+                })}
+              </p>
+            </Well>
+          )}
+        </ReviewSection>
+      )}
+
+      <section data-h2-margin-bottom="base(x3)">
+        <Heading level="h4" data-h2-margin="base(0, 0, x1, 0)">
+          {intl.formatMessage({
+            defaultMessage: "Sign and submit",
+            id: "+sWqVY",
+          })}
+        </Heading>
+        <div>
+          <FormProvider {...methods}>
+            <form onSubmit={methods.handleSubmit(handleSubmit)}>
+              <p data-h2-margin-bottom="base(x1)">
+                {intl.formatMessage({
+                  defaultMessage: `"You made it! By signing your name, you confirm that:"`,
+                  id: "d0KeTK",
+                })}
+              </p>
+              <ul>
+                <li>
+                  <p data-h2-margin-bottom="base(x.5)">
+                    {intl.formatMessage({
+                      defaultMessage: `"I’ve reviewed everything written in my application"`,
+                      id: "IFVdRD",
+                    })}
+                  </p>
+                </li>
+                <li>
+                  <p data-h2-margin-bottom="base(x.5)">
+                    {intl.formatMessage({
+                      defaultMessage: `"I understand that I am part of a community who trusts each other"`,
+                      id: "MKU7g2",
+                    })}
+                  </p>
+                </li>
+                <li>
+                  <p data-h2-margin-bottom="base(x.5)">
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "I promise that the information I’ve provided is true",
+                      id: "kAMYLw",
+                    })}
+                  </p>
+                </li>
+              </ul>
```

Comment:
This is more subjective but I found this reads oddly. It switches perspectives from "you" to "I" which seems odd. @mnigh Any opinion on the copy here? 🤔

Reply:
I agree, the pronoun switching seems odd. We use the first person singular statements in other places in the app wrapped in quotes (I'm not a fan), though I don't recall them switching from first to second person singular. I would think that changing the first person to second person singular would likely make the most sense, especially since rewriting the "You made it! By signing your name, you confirm that:" sentence would be awkward in the first person.

Reply:
I talked to Josh and he said to leave this for now until we figure out the pronoun prospective of the entire site.